utilities/pipelines/resourceRemoval/helper/Get-ResourceIdsOfDeployment.ps1

```diff
@@ -51,9 +51,9 @@ function Get-ResourceIdsOfDeploymentInner {
         'resourcegroup' {
             if (Get-AzResourceGroup -Name $resourceGroupName -ErrorAction 'SilentlyContinue') {
                 [array]$deploymentTargets = (Get-AzResourceGroupDeploymentOperation -DeploymentName $name -ResourceGroupName $resourceGroupName).TargetResource | Where-Object { $_ -ne $null }
-                foreach ($resourceId in ($deploymentTargets | Where-Object { $_ -notmatch '/deployments/' } )) {
-                    Write-Verbose ('Found resource [{0}]' -f $resourceId) -Verbose
-                    [array]$resultSet += $resourceId
+                foreach ($deployment in ($deploymentTargets | Where-Object { $_ -notmatch '/deployments/' } )) {
```

Comment:
Are changes to resource removal intended?

Reply:
Yes, the implementation stems from a conversation we had 'back in the day' about the removal - while I was working on the automanage PR. The ones in the PR are just those that did not make it into the revamp of the removal and are just cosmetic (i.e. more prepriptive + added output)

Reply:
I think `$deployment` was replaced by `$resourceId` by a previous PR. Maybe going back to `$deployment` is due to solved merge conflicts? Nevertheless, if that is a resourceId I'd keep that name

Reply:
There is a fair chance we walked back and forth a bit :D And I think we ended up with deployment - because - not all IDs actually `resourceIds`. Sometimes they do look like them but they're more like 'elementIds' or something that we cannot remove via 'Remove-AzResource'. What do you think? Don't have a strong opinion - I just recall we discussed 'something' :D

Reply:
It's a bit confusing in general as the function itself is called Get-ResourceIdsOfDeployment,the outer function refers to $deployments, the inner one refers to a mix of $deployments, $deploymentTargets, $resultSets. I think it's fine to call them $deployments for now but I'd open an issue to come up with a more intuitive naming (low prio)

Reply:
Updated several names :)